managed/src/main/java/com/yugabyte/yw/models/CertificateInfo.java

```diff
@@ -170,4 +176,21 @@ public static boolean isCertificateValid(UUID certUUID) {
   public boolean getInUse() {
     return Universe.existsCertificate(this.uuid, this.customerUUID);
   }
+
+  public ArrayNode getUniverseDetails() {
+    List<Universe> universes = Universe.universeDetailsIfCertsExists(this.uuid, this.customerUUID);
+    ArrayNode details = Json.newArray();
+    for (Universe universe : universes) {
+      UniverseDefinitionTaskParams universeDetails = universe.getUniverseDetails();
+      ObjectNode universePayload = Json.newObject();
+      universePayload.put("name", universe.name);
+      universePayload.put("updateInProgress", universeDetails.updateInProgress);
+      universePayload.put("updateSucceeded", universeDetails.updateSucceeded);
+      universePayload.put("uuid", universe.universeUUID.toString());
+      universePayload.put("creationDate", universe.creationDate.getTime());
+      universePayload.put("universePaused", universeDetails.universePaused);
+      details.add(universePayload);
+    }
```

Comment:
Can we add a helper function for both models in our common.Utils? Plus I am sure this is used somewhere else as well.